Doc/library/functions.rst

```diff
@@ -1063,10 +1074,10 @@ are always available.  They are listed here in alphabetical order.
 
     In all of the above cases, each call to ``locals()`` in a given frame of
     execution will return the *same* mapping object. Changes made through
-    the mapping object returned from ``locals()`` will be visible as bound,
-    rebound, or deleted local variables, and binding, rebinding, or deleting
-    local variables will immediately affect the contents of the returned mapping
-    object.
+    the mapping object returned from ``locals()`` will be visible as assigned,
```

Comment:
Since we have assignment statements rather than binding statements, I realised I should use different verbs in these docs.

Reply:
I imagine that 'bind' is still used here and there in the docs, but changing here while editing is fine.  Either way, users must learn that the joining action happens in a target namespace rather than in either object.

Reply:
Yeah, we definitely do use "binding" and "bound" in various places. It just looked weird here once I saw it in context with the rest of the builtin docs rather than standing on its own in PEP 558 and the pre-beta-1 PR diff.